core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/ResourceTypeConverter.java

```diff
@@ -43,10 +43,15 @@ public Resource toResource(ResourceDocument doc) {
 			return resourceTypeConverters.get(doc.getResourceType()).toResource(doc);
 		}
 		
+		public Map<String, ResourceTypeConverter> getResourceTypeConverters() {
+			return resourceTypeConverters;
```

Comment:
Make sure the Map returned here is immutable.

Reply:
Done